javascript/javascript3/week1/weatherApp/weatherApp.js

```diff
@@ -0,0 +1,74 @@
+
+const inputElement = document.getElementById('input-city');
+const btnElement = document.getElementById('weatherButton');
+
+btnElement.addEventListener('click', ()=> {
+
+    if(inputElement.value){
+        const cityName = inputElement.value.toLowerCase();
+        let key = '6284b3975ab711a2f7c25e4a1d0434a9';
```

Comment:
I would personally not use let here to define the key. I think there are a few different options you could use here instead.
1. Write a single function that holds all the details of the API call (more on this in a comment below), where you can define the API key with const (since you are not going to set it again)
2. Define the API key at the top of the file so it's available to all the other functions, a so-called global variable. In general, global variables are usually not the best idea (I'm sure you've heard this before, else I will happily elaborate).
3. The more advanced solution: This might be out of scope of this exercise, but if you want an extra challenge, you could check out how to set the API key as an environment variable using the built-in dotenv node package. This will allow you to hide away the API key so no other users can see your key on github!

Reply:
Yes i need to make them global variable, but yes iam really interested on the 3rd option that you have given , so that my key is encrypted